rc/filetype/markdown.kak

```diff
@@ -93,7 +93,7 @@ add-highlighter shared/markdown/inline/text/ regex "\H( {2,})$" 1:+r@meta
 
 define-command markdown-load-languages -params 1 %{
     evaluate-commands -draft %{ try %{
-        execute-keys "%arg{1}s```\h*\{?[.=]?\K\w+<ret>" # }
+        execute-keys "%arg{1}1s```\h*\{?[.=]?(\w+)\}?<ret>"
```

Comment:
this one is fine I think because it's obvious that the comment exists to balance the braces

Reply:
Getting rid of all three of them might make it possible to write an automated test that warns about the evil pattern at the cost of also fixing this one that's ugly but not so unclear.

(My audit of /lib/kak was very nearly automated and I'm hoping I can turn it into a fully automated check suitable to add to the test suite. I'm not quite there yet, and you do have to make some 'sanity' assumptions about the contents of kak files, but none that all the existing files don't already obey.)